pkg/lakom/verifysignature/verifier_test.go

```diff
@@ -121,12 +127,9 @@ IqozONbbdbqz11hlRJy9c7SG+hdcFl9jE9uE/dwtuwU2MqU9T/cN0YkWww==
 		var (
 			cache          verifysignature.SignatureVerificationResultCache
 			cachedVerifier verifysignature.Verifier
+			err            error
 		)
 		BeforeEach(func() {
```

Comment:
The `err` variable is not used later in this scope, hence it is better to declare it only in the function passed to `BeforeEach`.


```suggestion
		)
		BeforeEach(func() {
		var err error
```